examples/master/README.md

```diff
@@ -68,18 +68,27 @@ plugins and configuration needed.
 
 ### Sample Application
 
-The last step is to instantiate the `sample-app` template. The [sample
-app](sample-app) is a simple Ruby application
-that runs Sinatra and has one unit test defined to exercise the CI flow.
+The last step is to instantiate the `sample-app` templates.  Look for the `sample-app-masterslave-ci.json` and
+`sample-app-masterslave-stage.json` files under the [jenkins example folder in OpenShift Origin](https://github.com/openshift/origin/tree/master/examples/jenkins).
+These two templates revolve around a [simple Ruby application that runs Sinatra](https://github.com/mfojtik/sample-app) and has one unit test defined to exercise the CI flow.
```

Comment:
ick.  we should really be pointing to https://github.com/openshift/ruby-hello-world as a sample instead (but we'd need to add unit testing to it)
@mfojtik